### PR TITLE
i18n(es): Update of `why-astro.md`, `astro-components.md` and `astro-pages.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description. 
Updated as follows
- `why-astro.md` based on 5bc3f2d7cf9989114dbb170214f68c803bdbc2c1
- `astro-components.md` based on f5c9ed08f0d3e0099051429cc37013c8984d2ce2 and 77d098d089b65f8ddc21038d8bd17ca76f5de420
- `astro-pages.md` based on f3193c3df25fa52027e5b1047624a36a5439566f

### CODE REVIEW

File: src/pages/es/concepts/why-astro.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Por el contrario, la mayoría de los frameworks web modernos están diseñados p
 Esta es una de las diferencias más importantes que hay que entender sobre Astro. El enfoque especial de Astro en el contenido le permite a Astro hacer compensaciones y ofrecer características de rendimiento inigualables que no tendrían sentido para implementar de parte de otros frameworks web enfocados en aplicaciones.
 
 :::tip
-Si tu proyecto cae en el segundo campo de las "aplicaciones", es posible que Astro no sea la opción correcta para tu proyecto... **¡y está bien!** Consulta [Next.js](https://nextjs.org/) para una excelente alternativa de framework web enfocado en aplicaciones.
+Si tu proyecto cae en el segundo campo de las "aplicaciones", es posible que Astro no sea la opción correcta para tu proyecto... **¡y está bien!** Consulta [Next.js](https://nextjs.org/) para una alternativa a Astro más centrada en la aplicación.
```
Comment (kevinzunigacuellar) [inline src/pages/es/concepts/why-astro.md:29]:
```suggestion
Si tu proyecto cae en el segundo campo de las "aplicaciones", es posible que Astro no sea la opción correcta para tu proyecto... **¡y está bien!** Consulta [Next.js](https://nextjs.org/) para una alternativa a Astro más centrada en aplicaciones web.
```

File: src/pages/es/core-concepts/astro-components.md [hunk 2]
```diff
@@ -493,37 +500,9 @@ Para evitar agrupar el script, puedes usar el atributo `is:inline`.
...
-:::
-
-📚 Lea nuestra página de [referencias de directivas](/es/reference/directives-reference/#directivas-script--style) para más información acerca de las directivas disponibles para las etiquetas `<script>`.
-
-### Utilizando Scripts Externos
-
-**Cuándo utilizarlo:** Si tu archivo JavaScript vive dentro de la carpeta `public/`.
-
-Ten en cuenta que este enfoque saltea el procesamiento, compresión y optimización de JavaScript provista por Astro. (Para aprovechar todo esto debes utilizar el método `import` descripto en el siguiente item)
+Por defecto, Astro procesa y empaqueta las etiquetas `<script>`, agregando soporte para importar módulos npm, escribiendo TypeScript y más.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:503]:
```suggestion
Por defecto, Astro procesa y empaqueta las etiquetas `<script>`, agregando soporte para importar módulos npm, TypeScript y más.
```

File: src/pages/es/core-concepts/astro-components.md [hunk 3]
```diff
@@ -493,37 +500,9 @@ Para evitar agrupar el script, puedes usar el atributo `is:inline`.
...
-
-Astro detecta los módulos JavaScript importados del lado del cliente y luego comprime, optimiza y añade el JS a la página automáticamente.
-
-```astro
-// Importar módulo ESM
-<script>
-  import './algun-script-externo.js';
-</script>
-```
+Vea nuestra [Guía de scripting](/es/guides/client-side-scripts/) para más detalles.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-components.md:505]:
```suggestion
Vea nuestra [Guía de scripts del lado del cliente](/es/guides/client-side-scripts/) para más detalles.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias por la PR @dreyfus92 💜 deje unos comentarios

**Comment by dreyfus92** [comment]
Muchas gracias por el review @kevinzunigacuellar 🚀

**Pull request merged**